scripts/summarize_evidence.py

```diff
@@ -63,10 +63,11 @@ def buildTADkey(gwas_snp):
     output - The lookup info in the TAD gene dictionary
     i.e. [(Chromosome, TAD_ID:TAD_Start-TAD_End)
     """
+
     try:
         chrom = int(gwas_snp['chrom'])
     except:
-        chrom = gwas_snp['chrom'][3:]
+        chrom = gwas_snp['chrom'].replace('chr', '') 
 
```

Comment:
I don't like the current logic. "3" gets converted to an int but "chr3" gets converted to "3" as a string? Seems like you're begging for the this to break in the future.

Why not convert to string, then remove `chr`, and then try to convert to int? I'm not even sure you want the last step, as then chromosome X is a string but 3 is an int?

Reply:
> I don't like the current logic. "3" gets converted to an int but "chr3" gets converted to "3" as a string? Seems like you're begging for the this to break things in the future.

I agree - the previous logic was to enable the function to accept formats of both `3` and `chr3`. If it was unable to transform the input into an int, it must be of the format `chr3`. Of course, this is not the case for the `X` and `Y` chromosomes! which is the source of the bug. FWIW, `chrom` is converted into a string within this function

> Why not convert to string, then remove chr, and then try to convert to int? I'm not even sure you want the last step, as then chromosome X is a string but 3 is an int?

I think 6eb91607e3c78b9f56d8790b05089bf41f4a714f works as a simple solution (never try to convert to int)